contracts/src/protocol/OmniAVS.sol

```diff
@@ -9,53 +9,25 @@ import { IStrategy } from "eigenlayer-contracts/src/contracts/interfaces/IStrate
 import { ISignatureUtils } from "eigenlayer-contracts/src/contracts/interfaces/ISignatureUtils.sol";
 import { IServiceManager } from "eigenlayer-middleware/src/interfaces/IServiceManager.sol";
 
+import { IDelegationManager } from "../interfaces/IDelegationManager.sol";
 import { OmniPredeploys } from "../libraries/OmniPredeploys.sol";
-import { IOmniPortal } from "../interfaces/IOmniPortal.sol";
+import { OmniAVSStorage } from "./OmniAVSStorage.sol";
```

Comment:
Import this after all the interfaces, so it's clear there are definitions imported from this contract